src/pages/fieldManagement/settings/CustomFieldTable.jsx

```diff
@@ -62,6 +62,25 @@ export default function CustomFieldTable({
   );
   const fieldTypeName = fieldTypeDefinition.name;
 
+  const onViewCustomField = useMemo(
+    () => (_, field) => {
+      setPreviewInitialValue(field.defaultValue);
+      setPreviewField(field);
+    },
+    [],
+  );
+
+  const deriveEditHref = useMemo(
+    () => (_, field) =>
+      `/settings/fields/save-custom-field/${fieldTypeName}/${field.id}`,
+    [fieldTypeName],
+  );
+
+  const onDeleteCustomField = useMemo(
+    () => (_, field) => setDeleteField(field),
+    [],
+  );
```

Comment:
These can be simplified to `useCallback`s
> useCallback(fn, deps) is equivalent to useMemo(() => fn, deps)
https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
good point! i always forget about that one